doc/api/worker_threads.md

```diff
@@ -61,6 +61,125 @@ Worker threads inherit non-process-specific options by default. Refer to
 [`Worker constructor options`][] to know how to customize worker thread options,
 specifically `argv` and `execArgv` options.
 
+## `worker.connect(target[, data][, timeout])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `target` {number} The target thread id.
+* `data` {any} Any arbitrary, cloneable JavaScript value.
```

Comment:
Is there any reason `data` is supported here at all? I don't understand its function, wouldn't it be better to just pass whatever data over the MessagePort? (From an API design PoV: for this to do only one thing)

Reply:
It is mean to give an easy way for the `setConnectionListener` to identify the requesting thread to choose if to accept the connection or not, otherwise it would only rely on the thread id.